src/utils/reduceSQON.ts

```diff
@@ -1,16 +1,25 @@
 import {
 	CombinationKeys,
-	CombinationOperator,
 	FilterKeys,
-	FilterOperator,
-	SQON,
-	ScalarFilterKeys,
+	GreaterThanFilter,
+	InFilter,
 	isArrayFilter,
 	isFilter,
-} from '../types/sqon';
+	LesserThanFilter,
+	type CombinationOperator,
```

Comment:
in contrast, this works well, and naming convention matches my expectations.

truly hope you can understand my confusion...